database/index.js

```diff
@@ -0,0 +1,62 @@
+const mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/fetcher');
+const Promise = require('bluebird');
+
+var db = mongoose.connection;
+
+let destinationSchema = mongoose.Schema({
+  // TODO: your schema here!
+  id: {
+    type: Number, required: true, unique: true
+  },
+  name: String,
+  heart: Boolean,
+  reviews: Number,
+  averageRating: Number,
+  overview: String,
+  address: String,
+  toursAndExperiences: [String],
+  gallery: [String]
+
```

Comment:
looking at TripAdvisor, wondered if you also need to see data for open now, suggested duration, category, and #x of y things to do in (place) ?

Reply:
also noting, when the photos pop out, they also have a user and a comment, not sure if you're going to add that, but wanted to make sure to point it out!

Reply:
Good catch, I'm adding those in right now. 
 
For the:  #1 of 34 things to do in Mount Rainier National Park Mountains
I think I'm going to get the number of things to do by getting the length of the the array containing the names of tours/activities.